core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2309,7 +2309,7 @@ final class Stream[+F[_], +O] private[fs2] (private val free: FreeC[F, O, Unit])
   )(f: (O2, O3) => O4): Stream[F2, O4] =
     this.parZip(that).map(f.tupled)
 
-  /** Like `interrupt` but resumes the stream when left branch goes to true. */
+  /** Pause this stream when `pauseWhenTrue` emits `False`, resuming when `True` is emitted. */
```

Comment:
Hmm, I think this is actually the opposite:

```scala
@ val process = Stream(false) ++ Stream.sleep_(2.seconds) ++ Stream(true) ++ Stream.sleep_(5.seconds) ++ Stream(false)
process: Stream[IO[x], Boolean] = Stream(..)

@ Stream.awakeEvery[IO](1.second).debug().pauseWhen(process.debug()).compile.drain.timeout(10.seconds).unsafeRunSync()
false
1006586025 nanoseconds
true
2006826076 nanoseconds
<long delay here>
false
7002943065 nanoseconds
```

So it looks to me that the stream is indeed paused when `pauseWhenTrue` emits true. Which makes sense, given its name :)

Reply:
#1949